json/tbd_seb_n2.json

```diff
@@ -0,0 +1,47 @@
+{
+  "schema": "https://github.com/rd2/tbd/blob/master/tbd.schema.json",
+  "description": "testing JSON surface KHI entries",
+  "psis": [
+    {
+      "id": "good",
+      "parapet": 0.5,
+      "party": 0.9
+    },
+    {
+      "id": "compliant",
+      "rimjoist": 0.3,
+      "parapet": 0.325,
+      "fenestration": 0.35,
+      "concave": 0.45,
+      "convex": 0.45,
+      "balcony": 0.5,
+      "party": 0.5,
+      "grade": 0.45
+    }
+  ],
+  "khis": [
+    {
+      "id": "column",
+      "point": 0.5
+    },
+    {
+      "id": "support",
+      "point": 0.5
+    }
+  ],
+  "surfaces": [
+    {
+      "id": "Entryway  Wall 5",
+      "khis": [
+        {
+          "id": "column",
+          "count": 3
+        },
+        {
+          "id": "support",
+          "count": 4
+        }
+      ]
```

Comment:
Point thermal bridges (in W/K) are surface-specific entries, relying on KHI library items.

Reply:
So this is saying that the surface named "Entryway  Wall 5" is derated by two point bridges, a column and a support?

Reply:
Is the assumption that these point bridges are always to the outside environment?  E.g. a column or support inside a conditioned zone don't count as thermal bridges?

Reply:
1. In the example, the surface "Entryway Wall 5" has 3x columns + 4x supports (e.g. signage). As both columns and supports represent 0.5 W/K, that means that all 7x point thermal bridges represent a total of 3.5 W/K.
2. Your 2nd question is an important one at this stage. Although Topolys handles all geometry, TBD will currently only process linear and point thermal bridges affecting outside-facing walls, roofs and exposed floors. Partitions separating spaces with design (and operating) temperatures that differ by e.g. more than 10°C should be insulated (by code), and most likely should include minor and major thermal bridging. I'm adding this to the enhancement list.

Reply:
Cool, I missed the `count` property, that is a nice feature.